NodeJS/src/AgentClient.ts

```diff
@@ -44,10 +44,14 @@ export class AgentClient {
    * Create Agent and Station. Upon success, returns agent object and station object
    * @param agentUsername min length 2, max length 20, must pass ^[-.@\w]+$
    * @param agentPassword min length 8, max length 32, must have a uppercase character, must have at least one lowercase char, no whitespace, must contains a number, must contain one of ~!@?#$%^&*_
+   * @param firstName (optional) min length 2, max 16, must pass /^[a-zA-Z][a-zA-Z0-9_ .,'+]*$/
```

Comment:
Not seeing validation of first name and last name in code.

Reply:
thanks, will add

Reply:
done